packages/elements-react/src/util/ui/index.ts

```diff
@@ -98,7 +98,61 @@ type Entries<T> = {
   [K in keyof T]: [K, T[K]]
 }[keyof T][]
 
-export function useNodesGroups(nodes: UiNode[]) {
+/**
+ * Returns a list of auth methods from a list of nodes. For example,
+ * if Password and Passkey are present, it will return [password, passkey].
+ *
+ * Please note that OIDC is not considered an auth method because it is
+ * usually shown as a separate auth method
+ *
+ * This method the default, identifier_first, and profile groups.
```

Comment:
Something's missing here, right?

Reply:
```suggestion
 * Please note that OIDC is not considered an auth method because it is
 * usually shown as a separate auth method
 *
 * This method filters out the default, identifier_first, and profile groups.
```